vroom/python.vroom

```diff
@@ -24,7 +24,7 @@ surface the coverage results inside your vim editor.
   :silent edit blah.py
   :setf python
   % import sys<CR>
-  |if False:<CR>
+  |if 1 == 0:<CR>
```

Comment:
This part was interesting. Apparently newer versions of coverage.py are too clever and actually skip `if False:` blocks as not executable rather than calling them "uncovered", so there was a mismatch in reported coverage numbers for newer Coverage.py versions before I changed this.